docs/source/receptorctl/receptorctl_work_results.rst

```diff
@@ -0,0 +1,26 @@
+------------
+work results
+------------
+
+.. contents::
+   :local:
+
+``receptorctl work results`` Get results for previously or currently running unit(s) of work.
```

Comment:
```suggestion
``receptorctl work results`` get results for completed, or currently running, units of work.
```

Reply:
I feel like "previous" and "currently running" are two separate things so adding commas makes the two things distinct. Conjoining with or implies something was "previously running" which is more ambiguous to the final state. Does that include failed or aborted jobs?

Using "completed" here might make it a bit more clear that the job ran in the past. If it is the case that the command returns results for failed or aborted jobs, maybe we could flip things and say "currently or no longer running units of work". Not sure if that is great though. The best way might be really explicit but more verbose. "successfully completed, failed, stopped, or currently running units of work"

Reply:
The last is the winner, thank you for suggesting it